frontend-nannyway/src/components/header/AppBarSecondaryColor.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Grid from "@material-ui/core/Grid";
+import NavMenuSecondaryColor from "./NavMenuSecondaryColor";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+        backgroundColor: '#dfe2e7',
+    },
+    menuButton: {
+        marginRight: theme.spacing(2),
+    },
+    toolbar: {
+        minHeight: 72,
+        alignItems: 'flex-start',
+        paddingTop: theme.spacing(1),
+        paddingBottom: theme.spacing(2),
+    },
+}));
+
+export default function AppBarSecondaryColor() {
```

Comment:
I don't know what the differences between `<AppBarSecondaryColor>` and `<AppBarPrimaryColor>` are (I don't think it's the color), but if it's supposed to be the color you could just have a component called `<AppBar>` and pass a prop that changes the color.